fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/model/BulkExportJobInstanceRequest.java

```diff
@@ -84,106 +83,10 @@ public void setJobParameters(JobParameter jobParameters) {
         this.jobParameters = jobParameters;
     }
 
-    public static class JobParameter {
```

Comment:
same question... where did it go?

Reply:
added a comment related to this change in JobParameter.java

Reply:
common codes, so split out.